Dockerfile

```
@@ -0,0 +1,21 @@
+FROM python:3.12-slim
+
+RUN pip install poetry
+
+WORKDIR /code
+
+COPY README.md .
+
+COPY pyproject.toml poetry.lock* ./
+
+RUN poetry install --no-interaction --no-root
```

Устанавливай только основные зависимости - линтеры тебе не нужны
```suggestion
RUN poetry install --no-interaction --no-root --only main
```